docs/content/latest/troubleshoot/cluster/replace_failed_peers.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Fix 2/3 failed peers
+linkTitle: Fix 2/3 failed peers
+description: Fix 2/3 failed peers
+aliases:
+  - /troubleshoot/cluster/fix-2-3-failed-peers/
+  - /latest/troubleshoot/cluster/fix-2-3-failed-peers/
+menu:
+  latest:
+    parent: troubleshoot-cluster
+    weight: 835
+isTocNested: true
+showAsideToc: true
+---
+
+When a RAFT peer fails, YugabyteDB executes an automatic remote bootstrap to create a new peer from the remaining ones.
+If 2/3 of RAFT peers fail for a given tablet, we have to manually execute the equivalent of a remote bootstrap. We
+ can get a list of tablets in `yb-master-ip:7000/tablet-replication` yb-admin gui. 
+
+
+Assuming we have a cluster where:
+
+- a given tablet with UUID `TABLET1`
+- 3 tablet peers, 1 in good working order, referred to as `NODE_GOOD` and two broken peers, referred as `NODE_BAD1` and `NODE_BAD2`
+- We will be copying some tablet related data from the good peer to each of the bad peers
+
+These are the steps to follow in such scenario:
+
+- stop the bad TS, say `NODE_BAD1`, as we will be changing file system data underneath
+
+- on the `NODE_GOOD` TS, create an archive of the wals (raft data) and rocksdb (regular rocksdb) directories
+
+- copy these archives over to `NODE_BAD1`, on the same drive that `TABLET1` currently has its raft and rocksdb data
+
+- remove the old wals and rocksdb data for `TABLET1` from `NODE_BAD1`
+
+- unpack the data we copied over from `NODE_GOOD` into the corresponding (now empty) directories on `NODE_BAD1`
+
+- restart `NODE_BAD1`, so it can bootstrap `TABLET1` using this new data
+
+- restart `NODE_GOOD` so it can properly observe the changed state and data on `NODE_BAD1`
+
+- These steps need to be followed for both `NODE_BAD1` and `NODE_BAD2`
```

Comment:
also come to think of it, once 2 of them are in good working order, it's possible that's enough for the quorum to fix itself, eg: successfully remove `NODE_BAD2` through replicating a REMOVE_SERVER change config operation

Reply:
Can you paste exactly what to write in there ?

Reply:
What about RF 5 ? I'm assuming we need 3 in that case ?
 @bmatican

Reply:
I was thinking of removing this line and instead saying something like:
`At this point, NODE_BAD2 should be automatically fixed and removed from its quorum, as it has gotten a majority of health peers`

Reply:
As for RF5, maybe we should instead rephrase some things on the page to talk about `majority failure`, with `2/3` being an example, in the context of RF3.

What would change for RF5 for example, is that you'd need to bring up a majority of good peers. So if you've lost 3/5, you just need to fix 1 bad node. If you've lost 4/5, you need to fix 2 bad nodes. Main thing that would then change is, we still need to restart the good node(s), only after we've fixed the bad one(s).

Reply:
I'll do 5RF after commiting this.